sap/xssec/security_context.py

```diff
@@ -30,6 +31,25 @@ def _check_config(config):
         _check_if_valid(item, 'config.{0}'.format(prop))
 
 
+def _create_cert_and_key_files(cert_content, key_content):
+    cert_file = tempfile.NamedTemporaryFile(mode='w', delete=False)
```

Comment:
Is it really needed to create temporary files?
if so, okay.
But if there is a way to do it in memory I'd prefer this...
But I'm pretty sure you think the same way ;-)

Reply:
Yeah, the `httpx` library only accepts file paths to the key/cert file.